web/management/commands/create_test_data.py

```
@@ -574,4 +578,149 @@ def handle(self, *args, **kwargs):
                 )
             self.stdout.write(f"Created images for product: {product.name}")
 
+        # Create educational memes with random users
+        self.create_meme_test_data(subjects, users)
+
         self.stdout.write(self.style.SUCCESS("Successfully created test data"))
+
+    def create_meme_test_data(self, subjects, users):
```

Ruff (0.8.2)

Missing return type annotation for public function `create_meme_test_data`

Add return type annotation: `None`

(ANN201)